Lib/test/test_httpservers.py

```diff
@@ -1454,6 +1489,26 @@ def test_unknown_flag(self, _):
         self.assertEqual(stdout.getvalue(), '')
         self.assertIn('error', stderr.getvalue())
 
+    def test_response_headers_arg(self):
+        with mock.patch.object(HTTPServer, 'serve_forever'):
+            httpd = server._main(
+                ['-H', 'Set-Cookie', 'k=v', '-H', 'Set-Cookie', 'k2=v2', '8080']
+            )
+            request_handler_class = httpd.RequestHandlerClass
+            with mock.patch.object(
+                    request_handler_class, '__init__'
+            ) as mock_handler_init:
```

Comment:
```suggestion
            with mock.patch.object(
                request_handler_class, '__init__'
            ) as mock_handler_init:
```

Reply:
Fixed in 5f1fb944d41ea6604b2bab7ce9afd91ac33edabf